oximeter/oxql-types/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Core types for OxQL.
+
+use chrono::{DateTime, Utc};
+use std::time::Duration;
+
+pub mod point;
+pub mod table;
+
+pub use self::table::Table;
+pub use self::table::Timeseries;
+
+pub type TimeseriesKey = u64;
```

Comment:
This type is really about the data storage in ClickHouse. Ideally it'd be defined in `oximeter_db`, but I get why you need to move it down a level in the dependency tree. Maybe we could put it alongside the other schema-related types, like `TimeseriesSchema`, which I think as of #6355 is part of `oximeter_types::schema`?

Reply:
Sure, that sounds good to me.

Reply:
Done.